docs/ledger.md

```diff
@@ -8,13 +8,63 @@ At the core of a Ledger device there is a mnemonic seed phrase that is used to g
 Do not lose or share your 24 words with anyone. To prevent theft or loss of funds, it is best to keep multiple copies of your mnemonic stored in safe, secure places. If someone is able to gain access to your mnemonic, they will fully control the accounts associated with them.
 :::
 
+## Install the Cosmos Ledger application
+
+Installing the `Cosmos` application on your ledger device is required before you can use either [Lunie](#lunie-io-+-ledger-nano) or [`gaiacli`](gaia-cli-+-ledger-nano). To do so, you need to:
+
+1. Install [Ledger Live](https://shop.ledger.com/pages/ledger-live) on your machine. 
+2. Using Ledger Live, [update your Ledger Nano S with the latest firmware](https://support.ledger.com/hc/en-us/articles/360002731113-Update-device-firmware).
+3. On the Ledger Live application, navigate to the `Manager` menu . 
+    ![manager](./ledger-tuto-manager.png)
+4. Connect your Ledger Nano device and allow Ledger Manager from it. 
+5. On the Ledger Live application, Search for `Cosmos`. 
+    ![search](./ledger-tuto-search.png)
+6. Install the Cosmos application by clicking on `Install`.
+
+::: tip
+To see the `Cosmos` application when you search for it, you might need to activate the `Developer Mode`, located in the Experimental features tab of the Ledger Live application.
+
+![Devmode](./ledger-tuto-dev-mode.png)
+
+## Lunie.io + Ledger Nano  
+
+**Note: You need to [install the Cosmos app](#install-the-cosmos-ledger-application) on your Ledger Nano before using following this section**
+
+1. Connect your Ledger device to your computer, unlock it with the PIN and open the Cosmos app.
+2. Open [https://lunie.io](https://app.lunie.io/existing) in your web browser (latest version of Google Chrome preferred).
+3. Choose `Sign in with Ledger Nano S`.
+    ![lunie-option](./ledger-tuto-lunie-option.png)
+4. Click on `Sign in`. Do not forget to open the Cosmos app on your Ledger device for Lunie to recognize it. 
+
+That's it! You can now use Lunie with your Ledger Nano S. You will find your Cosmos address here:
+
+![lunie-address](./ledger-tuto-lunie-address.png)
+
+**Note: Each time you will send a transaction, you will need to confirm it on your Ledger device. Indication will be prompted from the Lunie interface**
+
+### (Optional) Confirm your address
```

Comment:
you can do this directly on the Cosmos Ledger app without relying on the CLI. Please add that instead